src/emulation_core/mips/datapath.rs

```diff
@@ -277,6 +277,11 @@ impl Datapath for MipsDatapath {
         *register = _data;
     }
 
+    fn set_fp_register_by_str(&mut self, _register: &str, _data: Self::RegisterData) {
+        let register = &mut self.coprocessor.registers[_register];
```

Comment:
This one's probably my fault from when I wrote the datapath trait, but can we remove the underscores in the argument names? I used them back in the datapath trait when I was writing to turn off the unused variable warnings, but they're not really needed anymore. There's a few other places in this PR that have underscored arguments that end up getting used too.